src/Microsoft.AspNetCore.Sockets.Client.Http/HttpConnection.cs

```diff
@@ -40,12 +42,13 @@ public class HttpConnection : IConnection
 
         private ReadableChannel<byte[]> Input => _transportChannel.In;
         private WritableChannel<SendMessage> Output => _transportChannel.Out;
+        private List<ReceiveCallBack> _callBacks = new List<ReceiveCallBack>();
```

Comment:
What if we just had just one callback (a Func) on HttpConnection? Just wondering when/if multiple callbacks are useful at this level...